### PR TITLE
make datatable editabe

### PR DESCRIPTION
**Pull request by shliachtx**

### CODE REVIEW

File: force-app/main/default/flexipages/tst.flexipage-meta.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jackdough) [inline force-app/main/default/flexipages/tst.flexipage-meta.xml:1]: @shliachtx  What's this file about?

File: force-app/main/default/lwc/datatable/datatable.js [hunk 2]
```diff
@@ -173,7 +177,7 @@ export default class Datatable extends LightningElement {
 
   @api
   get selectedRows() {
-    return this._selectedRows.map(row => { return row.charAt(0) === '/' ? row.slice(1) : row }); // remove prepended forward slash
+    return this._selectedRows;//.map(row => { return row.charAt(0) === '/' ? row.slice(1) : row }); // remove prepended forward slash
```
Comment (jackdough) [inline force-app/main/default/lwc/datatable/datatable.js:180]: remove commented code

File: force-app/main/default/lwc/datatable/datatable.js [hunk 3]
```diff
@@ -344,14 +348,15 @@ export default class Datatable extends LightningElement {
     columns = JSON.parse(JSON.stringify(columns))
       .map(col => {
         let fieldName = col.fieldName;
-        if (fieldName.endsWith('Id')) { // special case for salesforce relationship fields (this will not work for custom relationships)
-          fieldName = fieldName.replace('_Id', '.Name');
-          fieldName = fieldName.replace('Id', 'Name')
+        if (fieldName.endsWith('Link')) { // special case for salesforce relationship fields (this will not work for custom relationships)
```
Comment (jackdough) [inline force-app/main/default/lwc/datatable/datatable.js:351]: explain this? what's the Link vs. Id change?

File: force-app/main/default/flexipages/contact.flexipage-meta.xml [hunk 4]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jackdough) [inline force-app/main/default/flexipages/contact.flexipage-meta.xml:1]: Is there a reason to include flexipages for specific objects?

File: force-app/main/lwc-utils/lwc/tableServiceUtils/tableServiceUtils.js [hunk 5]
```diff
@@ -41,7 +41,7 @@ const applyLinks = (flatData) => {
   for (let row of dataClone) {
     Object.keys(row).forEach(key => {
       if (key.endsWith('Id')) {
-        row[key] = '/'+row[key];
+        row[key.replace('Id','Link')] = '/'+row[key];
```
Comment (jackdough) [inline force-app/main/lwc-utils/lwc/tableServiceUtils/tableServiceUtils.js:44]: @shliachtx is this just about making the property logical? (i.e. AccountId being a link doesn't make sense but AccountLink does)

**Review by jackdough — CHANGES_REQUESTED** [review]
@shliachtx , please see some comments inline. some changes need some more explanation.

File: force-app/main/lwc-utils/lwc/tableServiceUtils/tableServiceUtils.js (hunk 5 above)
Comment (shliachtx) [reply force-app/main/lwc-utils/lwc/tableServiceUtils/tableServiceUtils.js:44]: that, and leaving the Id field unchanged (for updates)

**Review by jackdough — APPROVED** [review]

**Pull request merged**